tasks/16_repeat-string/repeat-string.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * repeatStringNumTimes - repeats a given string num times
+ *
+ * @param {String} str
+ * @param {Number} num
+ * @return {String}
+ */
+export function repeatStringNumTimes(str, num) {
+  let result = '';
+  for (let i = 0; i < num; i++) {
```

Comment:
Look closer to `String` prototype, it has method for this case.

Reply:
Fixed